site/src/pages/docs/client-factory-builder.mdx

```diff
@@ -0,0 +1,164 @@
+# Building a client using ClientFactoryBuilder
+
+A <type://ClientFactoryBuilder> is a builder class that enables you to create a client factory
+which is able to customize various parameters. For example, some people want to make longer connection timeout.
+If so, the builder provides connectTimeoutMillis to change from the default value to what you want to change.
+
+You can simply create <type://ClientFactoryBuilder> as following example:
+
+```java
+try (ClientFactory factory =
+          ClientFactory.builder()
+                       // Set the connection timeout to 5 seconds.
+                       .connectTimeoutMillis(5000)
+                       // Disable certificate verification; never do this in production!
+                       .tlsNoVerify()
+                       .build()) {
+   final WebClient client = WebClient.builder()
+                                     .factory(factory)
+                                     .build();
+   // do something with web client
+  }
+```
+
+As above example, <type://WebClientBuilder> or <type://ClientBuilder> are able to use <type://ClientFactory>
+which is created from <type://ClientFactoryBuilder> using method `factory`. The <type://ClientBuilder> instance
+will use all parameters from the instance <type://ClientFactoryBuilder> that you defined.
+
+## What are available parameters?
```

Comment:
I'm not sure if we can maintain this well. How about just hyperlinking to `ClientFactoryBuilder` Javadoc so that a user can refer to the complete list of options?

Reply:
Yeah, that would be a good idea. I will replace the table as a link.